app/src/main/res/values/colors.xml

```diff
@@ -257,6 +263,7 @@
     <color name="select_login_header_normal_theme">@color/accent_bright_light_theme</color>
     <color name="mozac_widget_favicon_background_normal_theme">@color/mozac_widget_favicon_background_light_theme</color>
     <color name="mozac_widget_favicon_border_normal_theme">@color/mozac_widget_favicon_border_light_theme</color>
+    <color name="top_sites_border_normal_theme">@color/top_sites_border_light_theme</color>
```

Comment:
Can you move this to where we have `<!-- Top site colors -->` below around line 291.